gradle.properties

```
@@ -1,6 +1,6 @@
 # The --add-exports flags work around a bug with spotless and JDK 17
 # https://github.com/diffplug/spotless/issues/834
-org.gradle.jvmargs=-Xmx1g \
+org.gradle.jvmargs=-Xmx5g \
```

```suggestion
org.gradle.jvmargs=-Xmx1g \
```
Will commit this after testing

---

I change it to 2g locally. I'll make a separate PR.

---

Have you found a difference in speed in which it compiles between 2g and 5g locally?

---

We're stuck at 1g for now because Win32 can't address 2g. We can change that once we drop Win32 support after champs.

---

The amount of memory you give the JVM has no effect on the CPU usage.